indico/modules/attachments/clone.py

```diff
@@ -27,8 +27,9 @@ class AttachmentCloner(EventCloner):
     def is_available(self):
         return self._has_content(self.old_event)
 
-    def has_conflicts(self, target_event):
-        return self._has_content(target_event)
+    def get_conflicts(self, target_event):
+        if self._has_content(target_event):
+            return [_('The target event already has attachments')]
```

Comment:
```suggestion
            return [_('The target event already has materials')]
```

(we use "attachments" internally, but "materials" in user-facing text)